impl/src/main/java/org/apache/myfaces/cdi/model/FacesDataModelManager.java

```diff
@@ -58,10 +58,6 @@ public void addFacesDataModel(Class<?> forClass, Class<? extends DataModel> data
 
     public void init()
```

Comment:
we should add a reinit method here instead of modifiy the existing logic

Reply:
good call.